docs/configure-custom-values-scalardl-ledger.md

```diff
@@ -196,3 +174,18 @@ ledger:
   scalarLedgerConfiguration:
     ledgerLogLevel: INFO
 ```
+
+### Taints/Tolerations configurations (Optional based on your environment)
+
+If you want to control pod deployment using the taints and tolerations of Kubernetes, you can use `ledger.tolerations`.
+
+You can configure them using the same syntax as the tolerations of Kubernetes. So, please refer to the official document [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for more details on the tolerations configuration of Kubernetes.
```

Comment:
Some minor suggestions here. (Changes made here are consistent across other docs in this PR.)

```suggestion
If you want to control pod deployment by using the taints and tolerations in Kubernetes, you can use `ledger.tolerations`.

You can configure taints and tolerations by using the same syntax as the tolerations in Kubernetes. For details on configuring tolerations in Kubernetes, see the official Kubernetes documentation [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/).
```